aspnetcore/blazor/forms-validation.md

```diff
@@ -208,6 +209,19 @@ The `DataAnnotationsValidator` component doesn't validate subproperties of compl
 
 The <xref:System.ComponentModel.DataAnnotations.CompareAttribute> doesn't work well with the `DataAnnotationsValidator` component. The `Microsoft.AspNetCore.Blazor.DataAnnotations.Validation` package introduces an additional validation attribute, `ComparePropertyAttribute`, that works around these limitations. In a Blazor app, `ComparePropertyAttribute` is a direct replacement for the `CompareAttribute`. For more information, see [CompareAttribute ignored with OnValidSubmit EditForm (aspnet/AspNetCore \#10643)](https://github.com/aspnet/AspNetCore/issues/10643#issuecomment-543909748).
 
+### Validate nested models
+
+Blazor provides support for validating form input using data annotations with the built-in `DataAnnotationsValidator`. However, the `DataAnnotationsValidator` only validates top-level properties of the model bound to the form.
```

Comment:
While working through this in one of my personal app's over the weekend, I found there's an important detail missing from this section. You need to annotate your model properties with `[ValidateComplexType]` too. For example:

```csharp
public class LoanApplication
{
    [ValidateComplexType]
    public Applicant Applicant { get; set; } = new Applicant();

    [ValidateComplexType]
    public LoanAmount LoanAmount { get; set; } = new LoanAmount();
}
```

Reply:
Thanks ... I'll add that in a few minutes. I'm making a final pass now before calling for review on the weekend batch of PRs.

Reply:
btw --- *personal app*? ... `Applicant` ... `LoanAmount` ... mmmmmm ....

**_Scott Bank_** &ndash; *Your trusted partner in banking services!*:tm:

May I have a loan application? ... need a restored 1979 CJ-5!

Reply:
btw @scottaddie ... it's already there ...

> To validate these types, the `Microsoft.AspNetCore.Blazor.DataAnnotations.Validation` package introduces the `ValidateComplexType` validation attribute that works in tandem with the `ObjectGraphDataAnnotationsValidator` component.

... **BUT** ... you've prompted me to put an example in, and that's probably a really nice add here. Commit on the way ...............